test/test_site/_markbind/variables.md

```diff
@@ -10,3 +10,5 @@
 
 <span id="global_variable_overriding_included_variable">Global variable overriding included variable</span>
 <span id="included_global_variable">Global variable</span>
+<span id="page_global_variable">Global Variable</span>

```

Comment:
In context of the test that uses the variable, we can just use `included_global_variable` rather than declaring a separate `page_global_variable`?